src/dra/4A538.c

```diff
@@ -719,84 +719,81 @@ void ResetEntityArray(void) {
     }
 }
 
-#ifndef NON_MATCHING
-INCLUDE_ASM("dra/nonmatchings/4A538", RenderEntities);
-#else
 typedef struct {
-    /* 0x1F800034 */ u16 unk0;
-    /* 0x1F800036 */ s16 animSet;
-    /* 0x1F800038 */ s16 animCurFrame;
-    /* 0x1F80003A */ u16 w;
-    /* 0x1F80003C */ u16 h;
-    /* 0x1F80003E */ s16 x;
-    /* 0x1F800040 */ s16 y;
-    /* 0x1F800042 */ u16 xPivot;
-    /* 0x1F800044 */ u16 yPivot;
-    /* 0x1F800046 */ u16 unused12;
-    /* 0x1F800048 */ s32 index;
-    /* 0x1F80004C */ s32 unk18;
-    /* 0x1F800050 */ u32 unk1C;
-    /* 0x1F800054 */ s32 unk20;
-    /* 0x1F800058 */ s32 unk24;
-    /* 0x1F80005C */ OT_TYPE* ot;
-    /* 0x1F800060 */ POLY_GT4* poly;
-} EntitiesRenderer; // size=0x30
+    s16 unk0;
+    s16 animSet;
+    s16 animCurFrame;
+    u16 w;
+    u16 h;
+    s16 x;
+    s16 y;
+    s16 xPivot;
+    s16 yPivot;
+    u16 unused12;
+    s32 index;
+    s32 index2;
+    u32 spriteSheetIdx;
+    s32 facingLeft2; // set to 2 when facing left, 0 when not
+    s32 eDrawFlags;
+    u_long* ot;
```

Comment:
```suggestion
    OT_TYPE* ot;
```
This needs to get back to `OT_TYPE`, otherwise the graphics will look messed up